lib/streamlit/uploaded_file_manager.py

```diff
@@ -65,7 +82,12 @@ def _add_files(self, session_id, widget_id, files):
                 files = file_list + files
             self._files_by_id[files_by_widget] = files
 
-    def add_files(self, session_id, widget_id, files):
+    def add_files(
```

Comment:
Curious, is there any reason you wanted to type-annotate add_files but not the other methods (remove_files, update_file_count)? Did it seem useful in one case but not the other?

Reply:
Because I had modified the function earlier but then reverted it. Was too lazy to remove the typing and too lazy to add the typing for stuff I wasn't poking